app/views/admin/mods/index.html.erb

```diff
@@ -1,63 +1,58 @@
-<main>
-  <section class="row my-3" role="search">
-    <div class="col">
-      <ul class="nav nav-pills">
-        <li class="nav-item">
-          <%= link_to "General Community", admin_mods_path, class: "nav-link #{'active' if params[:state].blank?}" %>
-        </li>
-        <li class="nav-item">
-          <%= link_to "Tag Mods", admin_mods_path(state: :tag_moderator), class: "nav-link #{'active' if params[:state] == 'tag_moderator'}" %>
-        </li>
-        <li class="nav-item">
-          <%= link_to "Potential Mods", admin_mods_path(state: :potential), class: "nav-link #{'active' if params[:state] == 'potential'}" %>
-        </li>
-      </ul>
-    </div>
-    <div class="col">
-      <%= form_with url: admin_mods_path, method: :get, local: true, class: "form-inline justify-content-end" do |f| %>
-        <div class="form-group mx-sm-3">
-          <%= f.text_field :search, value: params[:search], class: "form-control", aria: { label: "Search" } %>
-          <%= f.hidden_field :state, value: params[:state] if params[:state].present? %>
-        </div>
-        <%= f.submit "Search", class: "btn btn-light" %>
-      <% end %>
+<div class="mb-6 flex items-center">
+  <ul class="nav nav-pills">
+    <li class="nav-item">
+      <%= link_to "General Community", admin_mods_path, class: "nav-link #{'active' if params[:state].blank?}" %>
+    </li>
+    <li class="nav-item">
+      <%= link_to "Tag Mods", admin_mods_path(state: :tag_moderator), class: "nav-link #{'active' if params[:state] == 'tag_moderator'}" %>
+    </li>
+    <li class="nav-item">
+      <%= link_to "Potential Mods", admin_mods_path(state: :potential), class: "nav-link #{'active' if params[:state] == 'potential'}" %>
+    </li>
+  </ul>
+
+  <%= form_with url: admin_mods_path, method: :get, local: true, class: "form-inline ml-auto" do |f| %>
+    <div class="form-group mx-sm-3">
+      <%= f.text_field :search, value: params[:search], class: "form-control", aria: { label: "Search" } %>
+      <%= f.hidden_field :state, value: params[:state] if params[:state].present? %>
     </div>
-  </section>
+    <%= f.submit "Search", class: "btn btn-light" %>
+  <% end %>
+</div>
 
-  <%= paginate @mods, theme: "internal" %>
+<%= paginate @mods, theme: "internal" %>
 
-  <table class="table">
-    <thead>
+<table class="crayons-table" width="100%">
+  <thead>
+    <tr>
+      <th>ID</th>
```

Comment:
**(question)**: @ludwiczakpawel I've seen scope=col removed, I understand those are for screen readers, so does adding it to the thead remove the need for the scope attr ?

Reply:
These shouldn't be removed. I'll bring them back.

Reply:
Cool I think they're in a couple of places 😊

Reply:
okeeeey, these should be back now..